src/api/app/models/bs_request.rb

```diff
@@ -88,6 +86,7 @@ class SaveError < APIError
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
   has_many :status_reports, as: :checkable, class_name: 'Status::Report', dependent: :destroy
+  has_many :target_project_objects, through: :bs_request_actions
```

Comment:
Associations yes but projects are not always project objects but strings :smile: The problem here is: In BsRequestAction there is an attribute target_project therefore we introduced the target_project_object association later on :scream: (You remember?). So this is to be 

* compliant with BsRequestAction
* otherwise we would need to specify the association name on the declaration of this association